onnxruntime/test/python/transformers/test_parity_mixtral_moe.py

```diff
@@ -124,7 +126,11 @@ def create_moe_onnx_graph(
     )
 
     model = helper.make_model(graph)
-    return model.SerializeToString()
+
+    model_path = "mixtral_moe.onnx"
```

Comment:
delete the model when all the tests are done

Reply:
So I would return model.SerializeToString() instead?

Reply:
you can use os.remove()

Reply:
Okay, I pushed both edits